src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
it will crash with `Cannot create intersection with not-fully defined Finder in configuration file.` exception if there is no finder configured in `.php_cs(.dist)?` file.